src/blob/index/core.rs

```diff
@@ -199,24 +212,24 @@ impl<FileIndex: FileIndexTrait> IndexStruct<FileIndex> {
         Ok((buf, bloom_offset))
     }
 
-    fn deserialize_filters(buf: &[u8]) -> Result<(Bloom, RangeFilter, usize)> {
+    fn deserialize_filters(buf: &[u8]) -> Result<(Bloom, RangeFilter)> {
         let (range_size_buf, rest_buf) = buf.split_at(size_of::<u64>());
         let range_size = deserialize(&range_size_buf)?;
         let (range_buf, bloom_buf) = rest_buf.split_at(range_size);
-        let bloom = Bloom::from_raw(bloom_buf)?;
+        let mut bloom = Bloom::from_raw(bloom_buf)?;
```

Comment:
It seems reasonable for `from_raw` to take an additional optional argument with an offset

Reply:
Done